docs/cards.md

```diff
@@ -0,0 +1,339 @@
+# Metaflow Cards
+
```

Comment:
Can you include a picture of a simple card?

Reply:
I also think it would be nice to have the recipe for a super-simple example e.g. a listing for DumbFlow, `python3 dumbflow.py run  --with card`, look at the output to get `abc/123` and `python3 dumbflow.py card view abc/123`. That way a novice can get a card working straight away, and then they can look at the more advanced stuff

Reply:
We can even make a GIF that shows this whole process. I will do that before final merge.